backend/src/pilstark/json_exporter/mod.rs

```diff
@@ -77,7 +77,12 @@ pub fn export<T: FieldElement>(analyzed: &Analyzed<T>) -> PIL {
             }
             StatementIdentifier::Identity(id) => {
                 let identity = &analyzed.identities[*id];
-                let file_name = identity.source.file.clone();
+                let file_name: String = identity
+                    .source
+                    .file
+                    .as_deref()
+                    .map(String::from)
+                    .unwrap_or_default();
```

Comment:
I think you could just extract the file name from the path here to make the tests work.